tools/oecert/gen_pubkey_header.sh

```diff
@@ -3,9 +3,18 @@
 # Copyright (c) Open Enclave SDK contributors.
 # Licensed under the MIT License.
 
+lock="/var/tmp/oecert_lock"
 destfile="$1"
 pubkey_file="$2"
 
+# Check if the lock exists
+if [ -f "$lock" ]; then
```

Comment:
Also check if $destfile exists. If so, exit. This prevents header files from being generated multiple times.

Reply:
Hmm.. there could be a potential issue if header file from old key lingers.